docs/choosing_clusters.md

```diff
@@ -15,6 +15,7 @@ Beginner Level:
 - [Kind](#kind)
 - [Docker for Desktop](#docker-for-desktop)
 - [Microk8s](#microk8s)
+- [Rancher Desktop](#rancher-desktop)
```

Comment:
i would probably put it in the Intermediate level, given that it's less widely used than k3d and uses a lot of the same base compontents